deep_translator/exceptions.py

```diff
@@ -123,7 +123,7 @@ class TooManyRequests(Exception):
 
     def __init__(
         self,
-        message="Server Error: You made too many requests to the server. According to google, you are allowed to make 5 requests per second and up to 200k requests per day. You can wait and try again later or you can try the translate_batch function",  # noqa
+        message="Server Error: You made too many requests to the server. According to google, you are allowed to make 5 requests per second and up to 200k requests per day. You can wait and try again later or you can try the translate_batch function",  # noqa: E501
```

Comment:
can you add some description about these tags `#noqa` at the end of each line? Why are they necessary?

Reply:
Those are flake8 errors code but that was there i Didn't add them just added the specific noqa code `E0501` which ignores the long line error.

Reply:
Yes I know you are basically ignoring the flake8 warning/error, but are these codes specific to flake8? 
Also I agree it would be better to fix this flake8 warning instead of ignoring it.

Reply:
@nidhaloff I have removed the error codes which are flake8 warning code and wrote exception messages into multiple lines. Thanks for your observations.